packages/t/tclap/xmake.lua

```diff
@@ -0,0 +1,28 @@
+package("tclap")
+    set_homepage("https://sourceforge.net/projects/tclap/")
+    set_description("This is a simple templatized C++ library for parsing command line arguments.")
+    set_license("MIT")
+
+    set_urls("https://netcologne.dl.sourceforge.net/project/tclap/tclap-$(version).tar.bz2")
+    add_versions("1.4.0-rc1", "33e18c7828f76a9e5f2a00afe575156520e383693059ca9bc34ff562927e20c6")
+
+    -- NOTE it could be useful to patch CMakeLists.txt
+    -- in order to avoid building examples
```

Comment:
we can also use `io.gsub` or `io.replace` to patch CMakeLists.txt. for example

```lua
packages/l/libtiff/xmake.lua
18:        io.replace("CMakeLists.txt", "add_subdirectory(contrib)", "", {plain = true})
19:        io.replace("CMakeLists.txt", "add_subdirectory(man)", "", {plain = true})
20:        io.replace("CMakeLists.txt", "add_subdirectory(html)", "", {plain = true})
21:        io.replace("CMakeLists.txt", "add_subdirectory(test)", "", {plain = true})
22:        io.replace("CMakeLists.txt", "add_subdirectory(tools)", "", {plain = true})
```

Reply:
Thank you, I will do that